cert/aziot-certd/src/lib.rs

```diff
@@ -180,11 +228,10 @@ impl UpdateConfig for Api {
         // Don't allow changes to homedir path or endpoints while daemon is running.
         // Only update other fields.
         let Config {
-            homedir_path: _,
             cert_issuance,
             preloaded_certs,
-            endpoints: _,
             principal,
+            ..
```

Comment:
The fields are explicitly listed intentionally, so that any refactoring of the config to add new fields has to be consciously handled here. That's why the aziotctl-config code also uses exhaustive patterns throughout, and why the comment on this line mentions why those fields are being ignored.

So please keep it as it was.